venue.html

```diff
@@ -27,19 +27,19 @@
     </div>
     
     <div id="content">
-        
+        <td><strong>Updated as of May 15</strong></td>
```

Comment:
Don't use `<td>` like this. It's only for cells in a `<table>`. Same for all `<td>` tags below

Just use `<p>` for paragraph